test/index.js

```diff
@@ -0,0 +1,35 @@
+var expect = require('chai').expect
+var path = require('path')
+var fs = require('fs')
+var url = require('url')
+
+describe('url-transformer.js', function () {
+  it('must export function and create output/paths.txt', function(done){
+
+    var urlTransformer = require(path.join(__dirname, '../url-transformer'))
+    var inputFile = path.join(__dirname, '..', 'data', 'urls.txt')
+    var outputFile = path.join(__dirname, '..', 'output', 'paths.txt')
+    fs.unlinkSync(outputFile)
```

Comment:
This will throw if the file doesn't exist -- we should wrap it in a try-catch and probably move it to a `before` block.